allennlp/modules/elmo.py

```diff
@@ -0,0 +1,422 @@
+import json
+
+from typing import Union, List, Dict, Any
```

Comment:
Can you group these imports into 3 blocks?

Reply:
done